1-org/envs/shared/projects.tf

```diff
@@ -14,6 +14,15 @@
  * limitations under the License.
  */
 
+locals {
+  hub_and_spoke_roles = [
+    "roles/compute.instanceAdmin",
+    "roles/iam.serviceAccountAdmin",
```

Comment:
Does network SA need service account admin?

Reply:
Yes, it does for the hub and spoke flow where it creates service accounts for the transitivity machines

https://github.com/terraform-google-modules/terraform-example-foundation/blob/9bd69dbc905eab48e359f415e98baa30b58dd18b/3-networks/modules/transitivity/main.tf#L21-L30